pvlib/spectrum/mismatch.py

```diff
@@ -531,26 +531,67 @@ def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
 
 def spectral_factor_sapm(airmass_absolute, module):
     """
-    Calculates the SAPM spectral loss coefficient, F1.
+    Calculates a module-specific SAPM spectral loss coefficient, :math:`f_1`,
+    equivalent to the spectral mismatch factor, using absolute
+    (pressure-corrected) air mass, :math:`AM_a`.
+
+    The SAPM spetral factor function is part of the broader Sandia Array
+    Performance Model, which defines five points on an IV curve using empirical
+    module-specific coefficients. Module coefficients for the SAPM are
+    available in the SAPM database and can be retrieved for use in the
+    ``module`` parameter through
+    :pyfunc:`pvlib.pvsystem.retrieve_sam()`. More details on the
+    SAPM can be found in [1]_, while a full description of the procedure to
+    determine the empirical model coefficients, including those for the SAPM
+    spectral correction, can be found in [2]_.
 
     Parameters
     ----------
     airmass_absolute : numeric
-        Absolute airmass
+        Absolute airmass [unitless]
+        Note: ``np.nan`` airmass values will result in 0 output.
 
     module : dict-like
-        A dict, Series, or DataFrame defining the SAPM performance
-        parameters. See the :py:func:`sapm` notes section for more
-        details.
+        A dict, Series, or DataFrame defining the SAPM performance parameters
+        See the :py:func:`pvlib.pvsystem.sapm` notes section for more details.
 
     Returns
     -------
-    F1 : numeric
+    :math:`f_1` : numeric
         The SAPM spectral loss coefficient.
```

Comment:
What is the correct way to format this as it does not render as intended in the docstring?

Reply:
I think sphinx interprets this text as a python variable name rather than general RST markup.  I suggest `f1` or just leaving it as `F1`.